src/test/java/com/exasol/adapter/document/DocumentAdapterIT.java

```diff
@@ -131,15 +132,12 @@ private static void writeCurrentVersionToMockProjectPom() throws IOException {
     void testUseMappingFromFile() throws Exception {
         final String content = JsonSample.builder().basic().withFields(JsonSample.ADDITIONAL_FIELDS).build();
         testSetup.getDefaultBucket().uploadStringContent(content, "mapping.json");
-        final VirtualSchema virtualSchema = createVirtualSchema("/bfsdefault/default/mapping.json");
-        try {
+        try (final VirtualSchema virtualSchema = createVirtualSchema("/bfsdefault/default/mapping.json")) {
```

Comment:
Ah - I see!
Now it pays off, having the test-db-builder [`DatabaseObject` ](https://github.com/exasol/test-db-builder-java/blob/b429113cf3a16bfba5f9821bb96231c9222f65d9/src/main/java/com/exasol/dbbuilder/dialects/DatabaseObject.java#L6) implement AutoClosable, right?

Reply:
These cases could also be done in `DocumentAdapterTest`, I saw unchanged, remove, and add there but still no "change data type".